enterprise/actions.py

```diff
@@ -58,3 +58,18 @@ def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argu
 
     export_as_csv.short_description = description
     return export_as_csv
+
+
+def get_clear_catalog_id_action(description=None):
+    """
+    Return the action method to clear the catalog ID for a EnterpriseCustomer.
+    """
+    description = description or "Unlink selected objects from existing course catalogs"
```

Comment:
@haikuginger mark "Unlink selected objects from existing course catalogs" for translation + don't forget to run `make extract_translations` and commit django.po / djangojs.po